### ISSUE TITLE
Never getting stake back problem for executors | Executor friendliness. 

### ISSUE THREAD

**Issue by hilmarx** [gelatodigital/gelato-network#74]
Let's assume there is only one executor left in the system, meaning it has no one to delegate its claims to.

Now it also wants to leave the system as it is somehow flawed. There are however still providers with a stake that have execution claims running. Now, this executor will as long as there are still execution claims left, never be able to withdraw its stake. 

Similar issue: Let's say we have a node running that can handle 1000 claims at the same time. Currently, there is no way, how we as executors could say: "No more than 1000 claims please", as we could get 2000 more tomorrow if a provider or executors simply reassigns the claims to us. This could also be done maliciously, lets say you are an executor and you want to be the only one, simply get a lot of claims minted and re-assign these to smaller executors who cant handle the load and watch them fail. This of course could turn out to be a wrong strategy if this node manages to scale up quite rapidly. 

In summary: If you are the last node standing, you are basically screwed. In practice, we will probably be that node and could also crank up the rent extremely high and the rent period super low to make it worthwhile for us.

So I would say let's worry about being more executor friendly in the next release. Our focus is and still should be providers.
